core/sql/sqlcomp/CmpSeabaseDDLindex.cpp

```diff
@@ -518,20 +521,13 @@ void CmpSeabaseDDL::createSeabaseIndex( StmtDDLCreateIndex * createIndexNode,
 
   ActiveSchemaDB()->getNATableDB()->useCache();
 
-  // save the current parserflags setting
-  ULng32 savedParserFlags = Get_SqlParser_Flags (0xFFFFFFFF);
-  Set_SqlParser_Flags(ALLOW_VOLATILE_SCHEMA_IN_TABLE_NAME);
-
   BindWA bindWA(ActiveSchemaDB(), CmpCommon::context(), FALSE/*inDDL*/);
   CorrName cn(tableName.getObjectNamePart().getInternalName(),
 	      STMTHEAP,
 	      btSchemaNamePart,
 	      btCatalogNamePart);
 
-  NATable *naTable = bindWA.getNATable(cn); 
-
-  // Save parser flags settings so they can be restored later
-  Set_SqlParser_Flags (savedParserFlags);
```

Comment:
Glad you took this out, as this code is incorrect. (It should have used Assign_SqlParser_Flags at line 534.)